### ISSUE TITLE
Accessibility Bug - Unable to swipe to next control after selecting the font. Voice over focus is in font selection only

### ISSUE THREAD

**Issue by sryarama** [ckeditor/ckeditor4#4079]
## Type of report

Bug

## Provide detailed reproduction steps (if any)
When VoiceOver is on in IPad screen reader focus trapped inside the combo panel even after selecting an item. I am able to repro the issue in CKEditor 4 demo page as well. Below are the repro steps:

1. Turn on Voice over
2. Hit the URL https://ckeditor.com/ckeditor-4/demo/ 
2. Swipe till  'Styles' combo in CK editor and double tap
3. Select any style by double tap.
4. After selection swipe and verify whether the voice over focus goes to next control 'Format' or not

Attached screen recording for your reference.

### Expected result
As per meaningful sequence voice over focus should come out of it after selection and by swiping it should go to next button.

### Actual result
Unable to swipe to next control after selecting the style. Voice over focus is in style selection only.

## Other details

* Browser: Safari (iPad)
* Version and Model: 13.3.1 iPad Pro (2nd generation)
* OS: iOS
* CKEditor version: CKEditor 4.11.4
* Installed CKEditor plugins: Font

[IMG_0242.zip](https://github.com/ckeditor/ckeditor4/files/4692035/IMG_0242.zip)

**Comment by jacekbogdanski** [comment]
@sryarama thanks for the report. Since most of us working remotely currently, we have limited access to an iPad to verify your issue. We will back to you once we will be able to test it on a real device.

**Comment by Dumluregn** [comment]
Hello, I'm unable to reproduce the issue.

I tested on iPad Air `12.4.6` and when following your scenario, after choosing the style from dropdown the focus moved properly back to the editor content and it was possible to move it between editor controls (see attached video). The difference may be caused by the devices or software, but currently I can't verify it.
[voiceover.zip](https://github.com/ckeditor/ckeditor4/files/4960279/voiceover.zip)

**Comment by Dumluregn** [comment]
Do you have a possibility to check if you are able to reproduce the issue on the older device?

**Comment by sryarama** [comment]
> Do you have a possibility to check if you are able to reproduce the issue on the older device?

unfortunately I don't have older device.

**Comment by Dumluregn** [comment]
That is unfortunate. Since we didn't manage to reproduce this issue I'm closing it, but when we gain access to a newer software we will get back to this.
